src/bb-locale/en_US/LC_MESSAGES/messages.po

```diff
@@ -1,12 +1,12 @@
-# en_US translation of BoxBilling Application
-# Copyright 2011-2021 BoxBilling <info@boxbilling.org>
+# en_US translation of FOSSBilling Application
+# Copyright 2011-2021 FOSSBilling <info@boxbilling.org>
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: 4\n"
 "POT-Creation-Date: 2015-02-27 08:07+0000\n"
 "PO-Revision-Date: 2021-08-10 09:00+0300\n"
-"Language-Team: BoxBilling TM <info@boxbilling.org>\n"
+"Language-Team: FOSSBilling TM <info@boxbilling.org>\n"
```

Comment:
Can the email be changed to info@fossbilling.org?

Reply:
Done

Reply:
> Can the email be changed to [info@fossbilling.org](mailto:info@fossbilling.org)?

Who is administering that email address?